addhost.php

```diff
@@ -18,16 +20,14 @@
 
 $options = getopt('Pbg:p:f::');
 
+$device = new Device;
+
 if (isset($options['g']) && $options['g'] >= 0) {
     $cmd = array_shift($argv);
     array_shift($argv);
     array_shift($argv);
     array_unshift($argv, $cmd);
-    $poller_group = $options['g'];
-} elseif (Config::get('distributed_poller') === true) {
-    $poller_group = Config::get('default_poller_group');
```

Comment:
Is this not needed before because it's set in `app/Console/Commands/DeviceAdd.php`?

Reply:
Correct, but in ValidateDeviceAndCreate it sets up default values.  So there is no need to set them before hand.